ckan/cli/search_index.py

```diff
@@ -28,9 +27,11 @@ def search_index():
                    u'ensures that changes are immediately available on the'
                    u'search, but slows significantly the process. Default'
                    u'is false.')
+@click.option('-c', '--clear', help='Clear the index', is_flag=True)
 @click.argument(u'package_id', required=False)
 def rebuild(
-        verbose, force, refresh, only_missing, quiet, commit_each, package_id
+        verbose, force, refresh, only_missing, quiet, commit_each, package_id,
```

Comment:
@steveoni We need to remove the `-r` / `refresh` option, as this is now the default behaviour